_locales/en/messages.json

```diff
@@ -49,6 +41,34 @@
         }
       }
     },
+    "importButton": {
+      "message": "Import",
+      "desription": "Button shown on the choose directory dialog which starts the import process"
+    },
+    "importChooserTitle": {
+      "message": "Choose directory with exported data",
+      "description": "Title of the popup window used to select data previously exported"
+    },
+    "importError": {
+      "message": "Unfortunately, something went wrong during the import. First, make sure all of the originally exported files are available. Then, please submit a debug log so we can help you get migrated!",
```

Comment:
Being mischievous, I picked the wrong directory and saw this. 

We could do more work to special case the likely failure scenarios...

e.g. if the file was renamed:

> Are you sure you picked the right file - SomeOtherDirectory doesn't look like a Signal Export 

And maybe before sending them to support:

> First, please export your data from Chrome again, and try importing the new export. If it still fails contact support...

These are just thoughts, take it or leave it.